geomstats/_backend/numpy/autodiff.py

```diff
@@ -18,7 +18,7 @@ def detach(x):
     return x
 
 
-def value_and_grad(func):
+def value_and_grad(*args, **kwargs):
```

Comment:
`value_and_grad(func, to_numpy=False)` could work too and would more closely follow the API of the others.

Reply:
The point is to get the wrong backend error instead of a possible error due to the arguments